discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -213,6 +213,21 @@ def wait_for_one_host_to_be_in_wrong_boot_order(self, fall_on_error_status=True)
             fall_on_error_status=fall_on_error_status,
         )
 
+    def wait_for_hosts_to_be_in_wrong_boot_order(
```

Comment:
instead of adding this method, please user this:
cluster.wait_for_one_host_to_be_in_wrong_boot_order
You can rename it to wait_for_atleast_one_host_to_be_in_wrong_boot_order, and pass node_count
it has fall_on_error_status, which is better in case of unexpected host error

Reply:
this function has also `fall_on_error_status`, am I missing something?
or you mainly want me to unify between these functions?

Reply:
missed that too. so just unify